js/sign/src/signers/node-crypto-signering-strategy.ts

```diff
@@ -0,0 +1,24 @@
+import crypto, { KeyObject } from 'crypto';
+import { isValidEd25519Key } from '../utils/utils.js';
+import { ISigningStrategy } from './signing-strategy-interface.js';
+
+// Class to be used when signing with parsed `crypto.KeyObject` private key
+// provided directly as `SignerOptions.key`.
```

Comment:
nit: `SignerOptions.key` is mentioned nowhere else. I would write something like "...provided directly in the contructor".